modules/041-linstor/templates/linstor-scheduler-admission/webhook.yaml

```diff
@@ -45,6 +45,7 @@ webhooks:
       - d8-descheduler
       - d8-ingress-nginx
       - d8-log-shipper
+      - d8-loki
```

Comment:
loki can use persistent volumes, please add it to exclusions of linstor-scheduler-admission webhook generator

https://github.com/deckhouse/deckhouse/blob/16d6f2c524e913e88fb356c0253c9dde4a3f9517/tools/linstor_scheduler_webhook/main.go#L32-L37